src/cypress/integration/Form/Checkbox/checkbox-with-other-options.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { checkboxCopy } from 'shared/data/copyContent';
+
+describe('checkbox-with-other-options-tests', () => {
+  before(() => {
+    cy.visitStorybook();
+    cy.loadStory('components-form-checkbox', 'with-other-options');
+  });
+
+  it('Should be displayed', () => {
+    cy.findByText(checkboxCopy.bestIceCream);
+  });
+
+  it('Should accept new values in "Other"', () => {
+    const expected = 'Strawberry';
+    cy.findByText(checkboxCopy.other)
+      .siblings('input')
+      .check({ force: true });
+
+    cy.get('.fd-other-option__input').type(expected);
+
+    cy.get('.fd-other-option__input').should('have.value', expected);
+  });
+
+  it('Should remove value from "Other" if unchecked', () => {
+    cy.get('.fd-other-option')
+      .siblings('input')
+      .uncheck({ force: true });
+
+    cy.get('.fd-other-option__input').should('not.have.value');
+  });
```

Comment:
Can you just leave a comment in here to update these once the `Checkbox` and `OtherOption` components are refactored? Something like:

```javascript
// TODO: Update to use findByLabel when Checkbox/OtherOption are refactored
```

This just helps to remind us to update it at some point, outside of the issue we made.